cmd/user-list/main.go

```diff
@@ -2,93 +2,61 @@ package main
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/alecthomas/kong"
 
+	"github.com/edulinq/autograder/internal/api/users"
+	"github.com/edulinq/autograder/internal/cmd"
 	"github.com/edulinq/autograder/internal/config"
-	"github.com/edulinq/autograder/internal/db"
 	"github.com/edulinq/autograder/internal/log"
-	"github.com/edulinq/autograder/internal/model"
 	"github.com/edulinq/autograder/internal/util"
 )
 
 var args struct {
 	config.ConfigArgs
 
-	Emails []string `help:"Optional list of users to limit the results to (leave empty for all users)." arg:"" optional:""`
-	Course string   `help:"Only include user enrolled in this course."`
-	Table  bool     `help:"Output data to stdout as a TSV." default:"false"`
+	Table   bool `help:"Output data to stdout as a TSV." default:"false"`
+	Verbose bool `help:"Use verbose output to show full request/response without specific formatting." default:"false"`
 }
 
 func main() {
 	kong.Parse(&args,
-		kong.Description("List users on the server (default) or from the specified course."),
+		kong.Description("List users on the server."),
 	)
 
 	err := config.HandleConfigArgs(args.ConfigArgs)
 	if err != nil {
 		log.Fatal("Could not load config options.", err)
 	}
 
-	db.MustOpen()
-	defer db.MustClose()
-
-	if args.Course == "" {
-		listServerUsers(args.Emails, args.Table)
-	} else {
-		listCourseUsers(args.Emails, args.Course, args.Table)
-	}
-}
-
-func listServerUsers(emails []string, table bool) {
-	users, err := db.GetServerUsers()
+	err = listServerUsers(args.Table)
 	if err != nil {
-		log.Fatal("Failed to get server users.", err)
-	}
-
-	if len(emails) > 0 {
-		newUsers := make(map[string]*model.ServerUser, len(emails))
-		for _, email := range emails {
-			newUsers[email] = users[email]
-		}
-
-		users = newUsers
-	}
-
-	if table {
-		fmt.Println(strings.Join(model.SERVER_USER_ROW_COLUMNS, "\t"))
-		for _, user := range users {
-			fmt.Println(strings.Join(user.MustToRow(), "\t"))
-		}
-	} else {
-		fmt.Println(util.MustToJSONIndent(users))
+		log.Fatal("Failed to list server users.", err)
 	}
 }
 
-func listCourseUsers(emails []string, courseID string, table bool) {
-	course := db.MustGetCourse(courseID)
+func listServerUsers(table bool) error {
```

Comment:
Your semantics here are off.
They do not match the semantics in cmd.PrintCMDResponse().

Instead of trying to recreate the semantics on every CMD that needs some custom printing,
I recommend just passing a function that handles printing the response content.
You have probably seen the "full" pattern that I have used before where I overload functions by calling the one with all the argument "full".
You can use this pattern and by default use a function that just outputs JSON (like you do in cmd.PrintCMDResponse() right now.